test/e2e/packetcapture_test.go

```diff
@@ -602,18 +610,21 @@ func runPacketCaptureTest(t *testing.T, data *TestData, tc pcTestCase) {
 		}
 	}
 
-	timeout := tc.pc.Spec.Timeout
-	if timeout == nil {
-		// It may take some time to upload.
-		timeout = ptr.To[int32](15)
-	}
-
-	if strings.Contains(tc.name, "timeout") {
-		// wait more for status update.
-		timeout = ptr.To[int32](*timeout + 5)
+	timeoutSeconds := tc.timeoutSeconds
+	// If timeout is not explicitly provided by test case...
+	if timeoutSeconds == 0 {
+		if tc.pc.Spec.Timeout != nil {
+			timeoutSeconds = int(*tc.pc.Spec.Timeout)
+		} else {
+			timeoutSeconds = 15
+		}
+		if strings.Contains(tc.name, "timeout") {
+			// wait more for status update.
+			timeoutSeconds += 5
+		}
 	}
```

Comment:
Probably to define a `defaultTimeoutSeconds`  and use it as following to make it clearer?

defaultTimeoutSeconds := 15
```suggestion
	timeoutSeconds := tc.timeoutSeconds
	// If timeout is not explicitly provided by test case...
	if tc.timeoutSeconds == 0 {
	    timeoutSeconds = defaultTimeoutSeconds
	}
	if tc.pc.Spec.Timeout != nil {
	   timeoutSeconds = int(*tc.pc.Spec.Timeout)
	}
	if strings.Contains(tc.name, "timeout") {
		// wait more for status update.
		timeoutSeconds += 5
	}
```

Reply:
I added `defaultTimeoutSeconds`, but I don't think the proposed `if` structure is really an improvement